test/fixtures/linter/projects/com.ui5.troublesome.app/webapp/controller/App.controller.js

```diff
@@ -20,6 +21,8 @@ sap.ui.define(["./BaseController", "sap/m/BackgroundDesign"],
 			// This should be detected, even when no module of sap.ushell is imported.
 			// Loading of the sap.ushell types should be triggered via the manifest.json dependency.
 			const oVariantSetAdapter = new sap.ushell.services.Personalization.VariantSetAdapter({});
+			const oVariantSetAdapter2 = new Personalization.VariantSetAdapter({});
```

Comment:
Importing modules of `sap/ushell` defeats the purpose of this test scenario described in the comment above. `sap.ushell.bootstrap` is a globally deprecated API which can be used in here as a replacement for `sap.ushell.services.Personalization.VariantSetAdapter`, which is now properly declared as a module, and not as a global anymore.